src/uct/sm/mm/base/mm_iface.c

```diff
@@ -57,6 +57,14 @@ ucs_config_field_t uct_mm_iface_config_table[] = {
      "Size of the FIFO element size (data + header) in the MM UCTs.",
      ucs_offsetof(uct_mm_iface_config_t, fifo_elem_size), UCS_CONFIG_TYPE_UINT},
 
+    {"FIFO_MAX_POLL", "16",
+     "Maximal number of receive completions to pick during RX poll",
+     ucs_offsetof(uct_mm_iface_config_t, fifo_max_poll), UCS_CONFIG_TYPE_ULUNITS},
+
+    {"FIFO_WINDOW_UPDATE_MAX_ITER", "128",
```

Comment:
`FIFO_WINDOW_UPDATE_RATE`?

Reply:
I forgot to push new changes, I re-named it to `FIFO_WINDOW_UPDATE_EPOCH`, what do you think works better?

Reply:
i think rate is better. afaiu, epoch is something bound to time boundaries, rather than frequency